scripts/stubsabot.py

```diff
@@ -32,7 +32,9 @@
 
 class ActionLevel(enum.IntEnum):
     def __new__(cls, value: int, doc: str) -> Self:
-        member = int.__new__(cls, value)
+        # Fails type-checking since updating "Self" types in #9694
+        # See: https://github.com/microsoft/pyright/issues/4668
+        member: Self = int.__new__(cls, value)  # pyright: ignore[reportGeneralTypeIssues]
```

Comment:
No longer necessary now the pyright bug's been fixed

```suggestion
        member = int.__new__(cls, value)
```